cli-toolchain/graphql.md

```diff
@@ -1694,41 +1686,64 @@ The `@function` directive generates these resources as necessary:
 
 ### @connection
 
-The `@connection` directive enables you to specify relationships between `@model` object types.
-Currently, this supports one-to-one, one-to-many, and many-to-one relationships. You may implement many-to-many relationships
-yourself using two one-to-many connections and joining `@model` type. See the usage section for details.
+The `@connection` directive enables you to specify relationships between `@model` types. Currently, this supports one-to-one, one-to-many, and many-to-one relationships. You may implement many-to-many relationships yourself using two one-to-many connections and a joining `@model` type. See the usage section for details.
+
+One can find a full schema example for a set of 17 access patterns at [docs/cli-toolchain/graphql#17-data-access-patterns](#17-data-access-patterns).
```

Comment:
Do we also have this in this PR or is that in the other one?

Reply:
Yes, this is in this PR.